code/modules/power/cell.dm

```diff
@@ -426,14 +426,17 @@
 	desc = "A rechargeable starch based power cell."
 	icon = 'icons/obj/service/hydroponics/harvest.dmi'
 	icon_state = "potato"
-	charge = 100
 	maxcharge = STANDARD_CELL_CHARGE * 0.3
 	charge_light_type = null
 	connector_type = null
 	custom_materials = null
 	grown_battery = TRUE //it has the overlays for wires
 	custom_premium_price = PAYCHECK_CREW
 
+/obj/item/stock_parts/cell/potato/Initialize(mapload, override_maxcharge)
+	charge = maxcharge * 0.3
+	. = ..()
```

Comment:
The physical `/obj/item/stock_parts/cell/potato` isn't what needs to be touched necessarily. (All this affects is maploaded / admin spawned potato batteries) 

`/datum/plant_gene/trait/battery` is what needs to be touched. It sets `maxcharge = seed.potency * 20` / `macharge *= 20` (for electrical activity).

Reply:
This also needs to be touched because its old value was a constant of `100`. But yeah I'll look into `/datum/plant_gene/trait/battery` as well

Reply:
Yeah the old value doesn't matter because it gets overridden *by * elec activity (`charge = maxcharge`), that's what I mean when I say it only affects mapload / admins pawned potatos. 

But it's still fine to change, it's harmless.

Reply:
Done